L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc

```diff
@@ -226,8 +220,8 @@ void PrimitiveSelection::process(
       int selected = map_tp_it->first;
       TriggerPrimitiveCollection& tmp_primitives = map_tp_it->second;  // pass by reference
 
-      int rpc_sub = selected / 6;
-      int rpc_chm = selected % 6;
+      int rpc_sub = selected / 8;
+      int rpc_chm = selected % 8;
```

Comment:
This works equally for regular RPCs and iRPCs?  Is "selected" a quantity that appears in the firmware?

Reply:
This is because I changed NUM_RPC_CHAMBERS from 7*6 to 7*8